src/NServiceBus.AcceptanceTests/Exceptions/Cant_convert_to_TransportMessage/When_cant_convert_to_TransportMessage_SuppressedDTC.cs

```diff
@@ -12,8 +13,13 @@ public class When_cant_convert_to_TransportMessage_SuppressedDTC : NServiceBusAc
         public void Should_send_message_to_error_queue()
         {
             Scenario.Define<Context>()
-                    .WithEndpoint<Sender>(b => b.Given(bus => bus.Send(new Message())))
-                    .WithEndpoint<Receiver>()
+                    .WithEndpoint<Sender>(b =>
+                    {
+                        b.Given(bus => bus.Send(new Message()));
+                        b.CustomConfig(configure => configure.SuppressOutdatedTimeoutDispatchWarning());
```

Comment:
isn't it possible to configure this option on the endpoint (in order to have it at the same place as the dtc suppression)?